python/src/dataset.rs

```diff
@@ -320,6 +321,27 @@ impl Operation {
         };
         Ok(Self(op))
     }
+
+    #[staticmethod]
+    fn create_index(
+        uuid: String,
+        name: String,
+        fields: Vec<i32>,
+        dataset_version: u64,
+    ) -> PyResult<Self> {
+        let new_indices = vec![Index {
+            uuid: Uuid::parse_str(&uuid).map_err(|e| PyValueError::new_err(e.to_string()))?,
+            name,
+            fields,
+            dataset_version,
+            fragment_bitmap: None,
```

Comment:
`fragment_bitmap` is essential for the maintenance of indices. We shouldn't leave it blank. You can set it to be a `set()` in Python.

Reply:
Now I added fragment_ids to the python api.

I guess in normal case, we should just pass the fragment_ids of all fragments to it for index built over a static dataset?

Reply:
> I guess in normal case, we should just pass the fragment_ids of all fragments to it for index built over a static dataset?

Yes that's accurate. There are other cases, such as when another process has appended during the index operation, or the index operation was only performed on a subset of data.